### PR TITLE
chore: Disable Mixpanel tracking

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

File: packages/frontend/src/mixpanel/index.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ let Mixpanel = {
     register: () => {}
 };
 
-if (BROWSER_MIXPANEL_TOKEN) {
+const shouldEnableTracking = false; // BROWSER_MIXPANEL_TOKEN
+
+if (shouldEnableTracking) {
     mixpanel.init(BROWSER_MIXPANEL_TOKEN);
```
Comment (agileurbanite) [inline packages/frontend/src/mixpanel/index.js:45]: What happens here? Do we send data to MixPanel?

Comment (Patrick1904) [reply packages/frontend/src/mixpanel/index.js:45]: Yep we send data to MixPanel if there's a token coming from the config

**Comment by followdarko** [comment]
@Patrick1904 just one question — why?

File: packages/frontend/src/mixpanel/index.js (hunk 1 above)
Comment (PinkiNice) [reply packages/frontend/src/mixpanel/index.js:45]: @Patrick1904 Are your trying to remove mixpanel tracking for wallet.near.org domains?

**Review by andy-haynes — APPROVED** [review]

**Pull request merged**